docs/README.md

```diff
@@ -393,17 +392,11 @@ git add; git commit -m "removing all gh-pages files"; git push -u origin gh-page
 git checkout main
 git pull
```

Comment:
checkout and pull of `main` is no longer needed, see how this is immediately followed by switching to a different branch?